### PR TITLE
Add "Works" and "About" Section 

### PR DESCRIPTION
**Pull request by Arch-Noize**
Added sections to the portfolio website. Using:
- Flexbox layout
- Grid layout
- Button interactions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,7 +37,61 @@ <h1>
         </div>
         <div class="buttons">
             <img src="/Images/Disabled.png"/>
-       </div>
+        </div>
+    </div>
+    
+    <div class="box">
```
Comment (christianbiring1) [inline index.html:43]: - [ ] Kindly make sure all the elements of this section(work section) are within a `section`  tag for semantics reasons. 
- [ ] You may also consider adding as many works as possible as is the case in the original template in Figma as you may see you only have one in your current implementation.

File: index.html [hunk 2]
```diff
@@ -37,7 +37,61 @@ <h1>
...
+        </section>
+        <button>
+            Get my resume
+        </button>
+    </div>
+    <div class="me-container">
+        <section class="languages">
+            <section class="logo">
+            
+            </section>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure all the content of your about section is grabbed inside one `section` tag. i.e. from About Myself up to `Skills` and not the way you have implemented it. you may consider checking [here](https://www.w3schools.com/tags/tag_section.asp) to know more about how to use a `section` tag.
- [ ] Also make sure you have `Frameworks` and `skills` parts on your page. as is the case on the original template. as you may notice you have only the `Language` part.
### Original template :point_down: 
image

File: index.html [hunk 3]
```diff
@@ -37,7 +37,61 @@ <h1>
         </div>
         <div class="buttons">
             <img src="/Images/Disabled.png"/>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure you use the correct image path for a better user experience as you notice how your image is not visible.
```suggestion
            <img src="./Images/Disabled.png"/>
            // Also as you notice there's no such image in your images directory
```
### Your design :point_down: 
image

### Original design :point_down: 
image

File: index.html [hunk 4]
```diff
@@ -37,7 +37,61 @@ <h1>
         </div>
```
Comment (christianbiring1) [inline index.html:37]: [OPTIONAL]
- [ ] Kindly make sure your design looks similar to the original template. as you may notice how different they are. sticking to the original design will give your page a great look and feel for a better user experience.
- [ ] Make sure your hamburger menu is placed on the right as is the standard for UX and better user experience on the mobile screen and also make sure the background image is well set as is the case in the original design.

### Your design :point_down: 
image

### Original template :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Arch-Noize , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
almost


## Highlights
- [ ] Good looking UI :art: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Arch-Noize) [reply index.html]: thats very weird thar that image specifically is not loading... everything else is loading?

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Arch-Noize , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle: 

You have submitted a project, that has failing linter checks. I would like to suggest that you fix the `stylelint` linter errors which are an indication that the `CSS` code you've written does not follow the agreed guidelines at Microverse. Kindly rectify this and request a review when you are ready.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by binhussen — APPROVED** [review]
Hi @Arch-Noize , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Fixed Request Changes✔️
- Well Structured Code✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**